testing/trino-testing/src/main/java/io/trino/testing/PlanDeterminismChecker.java

```diff
@@ -48,13 +48,10 @@ public void checkPlanIsDeterministic(String sql)
 
     public void checkPlanIsDeterministic(Session session, String sql)
     {
-        String previous = null;
+        String previous = planEquivalenceFunction.apply(getPlanText(session, sql));
```

Comment:
remove `previous = current;` below

Reply:
You're right, since they are all supposed to be identical, there's no point in tracking the previous values. Removed.